src/quo2/components/list_items/dapp/component_spec.cljs

```diff
@@ -0,0 +1,16 @@
+(ns quo2.components.list-items.dapp.component-spec
+  (:require [quo2.components.list-items.dapp.view :as dapp]
+            [test-helpers.component :as h]))
+
+(h/describe
+  "dapp test"
+  (h/test "renders default dapp"
+    (h/render [dapp/view
+               {:dapp     {:avatar nil
+                           :name   "Coingecko"
+                           :value  "coingecko.com"}
+                :state    :default
+                :action   :icon
+                :blur?    false
+                :on-press (fn [] (js/alert "Button pressed"))}])
```

Comment:
I think it's best to use a jest mock fn in these these tests

Reply:
Thanks for your review @J-Son89!

yes, done.